sergii.holubenko/task_2.rb

```diff
@@ -0,0 +1,2 @@
+ h = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+ h.count { |k,_| k.to_s.downcase.start_with?("yes") }
```

Comment:
this will not find keys that start with a space, like this: `' yesterday1 '=> 34`

Reply:
no new line in the end of the file